nhais-adaptor/adaptor/outgoing/fhir_helpers/tests/fixtures.py

```diff
@@ -0,0 +1,24 @@
+from adaptor.outgoing.fhir_helpers.operation_definition import OperationDefinitionHelper as odh

```

Comment:
I'm not sure this module makes sense in `fhir_helpers/tests`, as far as I can see, it's only used in `/adaptor/outgoing/tests`.

Reply:
👍 Done